app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt

```diff
@@ -240,6 +242,11 @@ object TestHelper {
         }
     }
 
+    fun getRunningTask(task: String) {
+        val activityManager = appContext.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
+        assertEquals(task, activityManager.appTasks[0].taskInfo.topActivity.toString())
+    }
+
```

Comment:
nice to have: a short comment about what this does and how to use it

Reply:
Done !